src/api/registerForEvents.js

```diff
@@ -48,6 +48,8 @@ const transform = (rawInitialData: RawInitialData, auth: Auth): InitialData => {
   return {
     ...rawInitialData,
 
+    realm_push_notifications_enabled_end_timestamp: 1706037926,
```

Comment:
This looks like it overrides the value from the server with this hard-coded value. Was this perhaps intended as temporary for local testing?

Reply:
Eep! Thanks for catching this; it was for local testing, yes.